pom.xml

```diff
@@ -11,13 +11,13 @@
       <dependency>
```

Comment:
In general: very good PR.
Please try to reduce the number of commits in future PRs (squash commits).